Makefile.toml

```diff
@@ -8,6 +8,7 @@ STACK_VERSION = { value = "latest", condition = { env_not_set = ["STACK_VERSION"
 CARGO_MAKE_CARGO_PUBLISH_FLAGS = "--dry-run"
 # RUST_BACKTRACE is set to "full" in cargo make's builtin makefiles/stable.toml
 RUST_BACKTRACE = { value = "0", condition = { env_not_set = ["RUST_BACKTRACE"]}}
+SECURE_INTEGRATION = { value = "true", condition = { env_not_set = ["SECURE_INTEGRATION"] }}
 OPENSEARCH_PROTOCOL_SUFFIX = { value = "s", condition = { env_true = ["SECURE_INTEGRATION"] }}
 OPENSEARCH_URL = { value = "http${OPENSEARCH_PROTOCOL_SUFFIX}://localhost:9200", condition = { env_not_set = ["OPENSEARCH_URL"] }}
```

Comment:
I think if `SECURE_INTEGRATION=false`, don't you get `"http${OPENSEARCH_PROTOCOL_SUFFIX}://localhost:9200"` as `OPENSEARCH_URL`? The value of `OPENSEARCH_PROTOCOL_SUFFIX` is not blank, it's not set.

Reply:
Didn't work for me :(